docs: 1.5 release (#8387)
langflow-ai/langflow: docs/docs/Components/components-helpers.md

</details>

## Message history

:::info
Prior to Langflow 1.5, this component was two separate components called **Chat History** and **Message Store**.
:::

> This component combines the **Chat History** and **Message Store** component into one component that can utilize Langflow's SQLite database, or connect to external memory, to store and retrieve chat messages.

Reviewer: ```suggestion
This component provides combined chat history and message store functionality. This component can use Langflow's SQLite database or external memory to store and retrieve chat messages.
```
